packages/react-data-grid/src/common/types.ts

```diff
@@ -34,52 +36,52 @@ export interface Column<V = unknown, DV = unknown> {
   /** Sets the column sort order to be descending instead of ascending the first time the column is sorted */
   sortDescendingFirst?: boolean;
   /** Editor to be rendered when cell of column is being edited. If set, then the column is automatically set to be editable */
-  editor?: React.ReactElement | React.ComponentType<EditorProps<V, DV>>;
+  editor?: React.ReactElement | React.ComponentType<EditorProps<R, V, DV>>;
```

Comment:
I think we may be able to provide better types for editors and formatters. It can probably be `typeof R[key]`